src/Features/CSharp/Portable/CSharpFeaturesResources.resx

```diff
@@ -524,4 +524,7 @@
   <data name="if_statement_can_be_simplified" xml:space="preserve">
     <value>'if' statement can be simplified</value>
   </data>
+  <data name="Convert_to_query" xml:space="preserve">
+    <value>Convert to query</value>
```

Comment:
this text does not seem language specific.  should it just be in FeatureResources?

Reply:
Added but not removed. Will remove in the next iteration

---
In reply to: [191058143](https://github.com/dotnet/roslyn/pull/27130#discussion_r191058143) [](ancestors = 191058143)